### DISCUSSION TITLE
macOS-like Column View (Miller Columns)

### DISCUSSION THREAD

**Discussion by egeesin** [spacedriveapp/spacedrive#1573] [Feature Request]
I'd love to have this feature which it helps me navigate and browse through complex directory structures. This would be one of the compelling reason for me to replace File Explorer of Windows (and it probably I would do the same if I were in a Linux distro)

Source: [Miller Columns - Wikipedia](https://en.wikipedia.org/wiki/miller_columns)

**Comment by utkubakir** [comment]
We have column view in our todo list :)
